crates/bevy_input/Cargo.toml

```diff
@@ -22,3 +22,4 @@ bevy_reflect = { path = "../bevy_reflect", version = "0.9.0-dev" }
 
 # other
 serde = { version = "1", features = ["derive"], optional = true }
+thiserror = "1.0"
```

Comment:
Note to other reviewers: we already have this in tree, so I'm fine with adding it here. It's a good crate!